src/profile/Index.vue

```diff
@@ -1,89 +1,171 @@
 <template>
   <BasePage>
     <template v-slot:header>
+      <!-- Todo update other locales -->
       {{ $vuetify.lang.t('$vuetify.profile.index.header', $root.idpConfig.idpName) }}
     </template>
+    
+    <ProfileProgress :profile="{user: $user, alternates, mfa}"/>
 
-    <v-row>
-      <v-col cols="6">
-        <Attribute :name="$vuetify.lang.t('$vuetify.profile.index.username')" :value="$user.idp_username" sameline />
-        <Attribute :name="$vuetify.lang.t('$vuetify.profile.index.lastLogin')" :value="$user.last_login | format" sameline />
-        <Attribute :name="$vuetify.lang.t('$vuetify.profile.index.manager')" :value="$user.manager_email" sameline />
-      </v-col>
-
-      <v-col cols="6">
-        <ProfileProgress :profile="{user: $user, alternates, mfa}"/>
-      </v-col>
-    </v-row>
+    <h3 class="mt-6">{{ $root.idpConfig.appName || 'Idp App'}} {{ $root.idpConfig.idpName }}</h3>
```

Comment:
Would it be better for `appName` to come form the skinney env or idpConfig?

Reply:
This seems more similar to the kinds of things in the skinny repo's `.env` file (similar to the color settings), and less like the things in the pw-api's idpConfig data (which are more like what password-store backends to use).

Reply:
I added `APP_DISPLAY_NAME` to the env, deleted the docker image, did a make but am still getting `undefined`. If I copy and paste another env it works, but not this one.🤔

Reply:
VUE_APP_DISPLAY_NAME worked